<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="map_name" default="/home/movel/.config/movel/maps/60e6871d8f8037719088e6c4" />
  <arg name="rgbd_depth_topic" default="/d435/depth/image_rect_raw" />
  <arg name="rgb_color_topic" default="/d435/color/image_raw" />
  <arg name="rgbd_camera_info" default="/d435/color/camera_info" />
                                                                      
  <include file="$(find orbomator)/launch/rgbd_loc.launch">
    <arg name="map_name" value="$(arg map_name)"/>
    <arg name="rgb_color_topic" value="$(arg rgb_color_topic)"/>
    <arg name="rgbd_depth_topic" value="$(arg rgbd_depth_topic)"/>
    <arg name="rgbd_camera_info" value="$(arg rgbd_camera_info)"/>
  </include>
  <include file="$(find orbomator)/launch/orbomator_transform.launch" />
</launch>